docs/publish-module/ios-specific-publish-flows/app-information-app-store.md

```diff
@@ -0,0 +1,35 @@
+---
+title: App Information from App Store
+metaTitle: App Information from App Store
+metaDescription: App Information from App Store
+sidebar_position: 1
+---
+import Screenshot from '@site/src/components/Screenshot';
+
+# App Information from App Store
+
+This step allows you to see app version information from TestFlight and the App Store on the same screen, along with the version you want to submit. When the step runs, your latest version information in TestFlight and App Store will be displayed as follows.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2917-infoDetail.png' />
```

Comment:
I think there should be more information about the app headers in this image:

- Bundle ID
- App Icon
- App Name
- Version
- Build Number
- Uploaded Date
- Expire Date
- Release Type
- Proccessing State


For example
- App Icon: Specifies the icon that will appear for the app in the selected platform. Check Apple's [App icons](https://developer.apple.com/design/human-interface-guidelines/app-icons) page for details.

---
@csonuryilmaz do you agree with that or is it unnecessary?

Reply:
I agree @ftmzhrsckldr. :+1: 

It will be good to explain the app headers in the screenshot.